src/interactivity/base/ServiceLocator.cpp

```diff
@@ -100,28 +100,33 @@ void ServiceLocator::RundownAndExit(const HRESULT hr)
     return status;
 }
 
-#pragma endregion
-
-#pragma region Set Methods
-
-[[nodiscard]] NTSTATUS ServiceLocator::SetAccessibilityNotifier(_In_ std::unique_ptr<IAccessibilityNotifier>&& notifier)
+[[nodiscard]] HRESULT ServiceLocator::CreateAccessibilityNotifier(const bool isPty)
 {
-    if (s_accessibilityNotifier)
+    if (isPty)
     {
-        NT_RETURN_NTSTATUS(STATUS_INVALID_HANDLE);
+        return S_FALSE;

```

Comment:
If this returns `S_FALSE`, doesn't the caller return a failure when it checks with `RETURN_IF_FAILED`? Do we want that?

Reply:
Ah, `S_FALSE` is a _success_ code :smile:

Reply:
Yeah it's strange. Also it's gone now.